google/cloud/bigtable/internal/bigtable_channel_refresh_test.cc

```diff
@@ -71,6 +72,59 @@ TEST(BigtableChannelRefresh, Disabled) {
       cq, Options{}.set<bigtable::MaxConnectionRefreshOption>(ms(0)));
 }
 
+TEST(BigtableChannelRefresh, Continuations) {
+  using ms = std::chrono::milliseconds;
+  using ns = std::chrono::nanoseconds;
+  using ::testing::ReturnRef;
+
+  promise<StatusOr<std::chrono::system_clock::time_point>> p;
+
+  auto mock_cq = std::make_shared<MockCompletionQueueImpl>();
+  grpc::CompletionQueue grpc_cq;
+  EXPECT_CALL(*mock_cq, cq).WillRepeatedly(ReturnRef(grpc_cq));
+
+  ::testing::InSequence s;
+  EXPECT_CALL(*mock_cq, MakeRelativeTimer)
+      .WillOnce([&p](std::chrono::nanoseconds duration) {
+        EXPECT_LE(ns(ms(500)).count(), duration.count());
+        EXPECT_LE(duration.count(), ns(ms(1000)).count());
```

Comment:
Does `.count()` need to be involved?  That is, can't we compare the durations directly?

Reply:
We could compare the durations directly. I think the idea was to have (arguably) better error messages. e.g.
```
Expected: (ns(ms(1000)).count()) <= (duration.count()), actual: 1000000000 vs 852000000
```
vs.
```
Expected: (ms(1000)) <= (duration), actual: 8-byte object <E8-03 00-00 00-00 00-00> vs 8-byte object <00-8E AB-25 00-00 00-00>
```

Reply:
Fair enough.  Perhaps consider these options:

1. Leave it as-is.
2. Wrap each side of duration expectations in `absl::FromChrono`, so you even get even nicer output, including units.

Either of these (`.count()` or wrapping) could disappear when durations become streamable in C++20.